spec/models/task_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Task, type: :model do
+
+  let(:task_a) { FactoryBot.create(:task) }
```

Comment:
`FactoryBot.`という記載はRSpecの設定ファイルに設定を追記することで省略できます。
```suggestion
  let(:task) { create(:task) }
```